app/src/main/java/io/github/dmi3coder/scorsero/score/ScoreStarterController.kt

```diff
@@ -41,6 +40,7 @@ class ScoreStarterController() : Controller(), ScoreCreationContract.View, OnCli
 
       override fun onStateChanged(bottomSheet: View, newState: Int) {
         if (newState == BottomSheetBehavior.STATE_HIDDEN) {
+          view!!.title_field.clearFocus()
```

Comment:
It is possible that `view` will be null in this situation, use `bottomSheet` instead